scala/private/rules/scala_test.bzl

```diff
@@ -57,6 +57,9 @@ _scala_test_attrs = {
     "colors": attr.bool(default = True),
     "full_stacktraces": attr.bool(default = True),
     "jvm_flags": attr.string_list(),
+    "reporter_class": attr.string(
```

Comment:
Looks amazing! 

Sorry for post merge CR but I have a small comment!

I'd use Bazel's `doc` attribute to document this new attribute.
I'm guessing one day we would want to auto-generate full manual and I think reverse engineering and by git blame is not the best way to go.